### ISSUE TITLE
Only ship colorized logs with SDK distros

### ISSUE THREAD

**Issue by gbbirkisson** [enonic/xp#8908]
We have a problem that was introduced with colorized logs. The problem is that we parse some fields out of the logs using logstash. When colors in logs were introduced this broke. Also some tools used in the terminal do not play well with colors.

The suggestion is to keep the colors for SDK distros but remove them from server distros.

**Comment by rymsha** [comment]
Only console logs are colorized by default
File logs are not.

Logstash is usually not the one collecting logs from console, only if you use "pipe" magic.

Also, we don't force log pattern/format: it is configurable in logback.xml

**Comment by gbbirkisson** [comment]
File logs are not shipped when using docker, the console logs are the ones being shipped. In modern architecture, console logs are picked up by docker, that are picked up by a log collector (filebeat) and shipped to logstash or elasticsearch. Similar thing is true with the docker syslog driver. Doing it this way minimizes configuration and enables us to use labels in docker and k8s to say what logs to ship. This is standard practice, so having colorized logs in the docker image by default causes problems for anybody using this kind of setup.

**Comment by rymsha** [comment]
related to #8638

**Comment by gbbirkisson** [comment]
Not really, because we are using the docker log system that relies on the console logs. In the case of K8s for example, any other approach would be stupid.

**Comment by rymsha** [comment]
wouldn't it be a problem anyway? For instance https://github.com/moby/moby/issues/22920

**Comment by gbbirkisson** [comment]
Multiline logs would always have to be handled by a some sort of a parser like logstash. It is very understandable that the folks at docker are not going to support that out of the box. 

As nice colorized logs are for development and debugging, they are a nuisance when it comes to log shipping simply because it introduces these escape characters in the log:

```
2021-05-05 12:32:54,052 ESC[34mINFO ESC[0;39m ESC[36mc.e.x.l.i.framework.FrameworkServiceESC[0;39m - Starting Enonic XP...
2021-05-05 12:32:54,679 ESC[34mINFO ESC[0;39m ESC[36mc.e.x.l.i.p.ProvisionActivatorESC[0;39m - Installing 94 bundles...
2021-05-05 12:32:56,525 ESC[34mINFO ESC[0;39m ESC[36mROOTESC[0;39m - bundle org.apache.felix.scr:2.1.24 (10) Starting with globalExtender setting: false
2021-05-05 12:32:56,559 ESC[34mINFO ESC[0;39m ESC[36mROOTESC[0;39m - bundle org.apache.felix.scr:2.1.24 (10)  Version = 2.1.24
```
